tor/torrc

```diff
@@ -2,9 +2,20 @@
 SocksPort   10.11.5.1:29050
 ControlPort 10.11.5.1:29051
 
-# Dashboard Hidden Service
+# Dashboard Hidden Service - v3
 HiddenServiceDir /var/lib/tor/web
+HiddenServiceVersion 3
 HiddenServicePort 80 10.11.0.2:80
 
+# Bitcoind Hidden Service P2P - v3
+HiddenServiceDir /var/lib/tor/bitcoin-p2p-v3
+HiddenServiceVersion 3
+HiddenServicePort 8333 10.11.1.1:8333
+
+# Bitcoind Hidden Service P2P - v2
+HiddenServiceDir /var/lib/tor/bitcoin-p2p-v2
+HiddenServiceVersion 2
```

Comment:
I don't see any harm adding both

Reply:
What is the benefit in supporting v2? Are there any services that require it?

If not, It seems like it could just cause extra confusion for no real benefit reason.

v3 is superior and what we're using for everything else, so I think it makes more sense to just stick to that.

Reply:
so we need to stick with v2 for p2p... v3 does not seem to work at all

Reply:
as in bitcoind fails to "see" it under ```externalip=```

Reply:
https://blog.torproject.org/v2-deprecation-timeline 😕

Reply:
well we have a problem then because v3 isn't supported with ```externalip=``` 

who wants to make a bug on the bitcoin project

Reply:
For now we can just add our own v3 hidden service, configure `bitcoind` to use the Tor proxy port for outgoing onion connections and remove the control port setting.

`bitcoind` won't be aware that it's exposed over Tor and won't broadcast it, but it will still accept incoming connections so will still be available remotely over Tor from software explicitly configured to connect to it.